Documentation/ceph-object.md

```diff
@@ -49,11 +49,11 @@ spec:
 
 After the `CephObjectStore` is created, the Rook operator will then create all the pools and other resources necessary to start the service. This may take a minute to complete.
 
-```bash
-# Create the object store
+```console
+## Create the object store
```

Comment:
not necessary
(i'll stop commenting on these, if you could review all the changes from # to ##)